proposals/2020-10-28-Experimental.md

```diff
@@ -113,12 +113,13 @@ This kind of requirement is handled for TypeScript libraries by [api-extractor](
 ### Unanswered Questions
 
 - If an operation is not marked as experimental, but it is using a schema which is (i.e. as its request object), then it is implicitly also unstable. Would this usage be considered invalid?
```

Comment:
In the request object example only requests using an experimental feature are potentially unstable, so I'd see this as a valid combination.